skbio/stats/composition.py

```diff
@@ -2058,3 +2061,407 @@ def dirmult_ttest(
         "Reject null hypothesis",
     ]
     return res[col_order]
+
+
+def _type_cast_to_float(df):
+    """Attempt to cast all of the values in dataframe to float.
+
+    This will try to type cast all of the series within the
+    dataframe into floats.  If a column cannot be type casted,
+    it will be kept as is.
+
+    See Also
+    --------
+    differential.regression.mixedlm
```

Comment:
Let's remove this, since everything has been ported from that differential package